### PR TITLE
Issue #3619: NeedBraces need clear description of allowSingleLineStatement

### PR DESCRIPTION
**Pull request by MohanadKh03**
Fixes: #3619
Continuation of #14133

### CODE REVIEW

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/blocks/needbraces/Example1.txt [hunk 1]
```diff
@@ -0,0 +1,29 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="NeedBraces"/>
```
Comment (romani) [inline src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/blocks/needbraces/Example1.txt:4]: Please remove such txt files.
They are not used in template file.

We already migrated this Check to java examples

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/blocks/needbraces/Example1.txt (hunk 1 above)
Comment (romani) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/blocks/needbraces/Example1.txt:4]: We need update in exampleX.java to show such attention point.

**Comment by MohanadKh03** [comment]
GitHub, generate website

**Comment by romani** [comment]
GitHub, generate website

**Review by romani — APPROVED** [review]
Awesome.

Thanks a lot for moving this across finish line.

**Pull request merged**